srcpkgs/libcamera/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'libcamera'
+pkgname=libcamera
+version=0.0.3
```

Comment:
There is a 0.0.4 available by now.

Reply:
Thanks! I updated the source and the package version.